Rocket.Chat/Controllers/Chat/ChatSections/MessageSection.swift

```diff
@@ -56,6 +63,10 @@ final class MessageSection: ChatSection {
             cell.delegate = self
         }
 
+        if let cell = cell as? SequentialMessageCell {
+            cell.delegate = self
+        }
+
```

Comment:
What about always having a delegate in the cells, or perhaps checking the negative states, like date separator and loader instead?

Reply:
@rafaelks I also don't like the way it is today, but until we use `ChatMessageProtocol` I think that it's best to leave this way.

In the near future, after we get everything working, I'm planning to split `ChatMessageCellProtocol` and its functionality.